pkgs/development/interpreters/clojure/nbb/deps.edn

```diff
@@ -0,0 +1,15 @@
+
+{
+ :deps {
+        com.github.seancorfield/honeysql {:git/tag "v2.2.891" :git/sha "796c734"}
```

Comment:
If clojure CLI is invoked (I supposed the `bb release` command is invoking it), clojure will make a network request to resolve the tag. To avoid that, remove `:git/tag` and use the full sha

Reply:
This is a bug and tracked upstream at https://clojure.atlassian.net/browse/TDEPS-223.

In case that makes you feel any better about doing the workaround :)